python/demo/demo_stokes.py

```diff
@@ -275,8 +275,11 @@ def nested_iterative_solver():
     # `scatter_forward`.
     with XDMFFile(MPI.COMM_WORLD, "out_stokes/velocity.xdmf", "w") as ufile_xdmf:
         u.x.scatter_forward()
+        P1 = element("Lagrange", msh.basix_cell(), 1, rank=1)
```

Comment:
Instead of using XDMFFile, should we use `VTX`, as this clearly reduces the accuracy of the solution?

Reply:
Could be done in a later PR. This change doesn't change the precision of the output, just makes it obvious rather than hidden.

ADIOS2 is an optional dependency, so there is a question as to how much it should be used in demos.

Reply:
I added comment to the XDMF function docstring pointing to `VTXWriter`.